src/app/components/CollapsableBookList.tsx

```diff
@@ -0,0 +1,56 @@
+import * as classNames from 'classnames';
+import { take } from 'lodash';
+import * as React from 'react';
+import MediaQuery from 'react-responsive';
+
+import { Icon } from '@ridi/rsg';
+import { ConnectedInlineHorizontalBookList } from 'app/components';
+import { Book } from 'app/services/book';
+import { Expander } from 'app/services/book/components/Expander';
+
+interface CollapsableBookListProps {
+  className: string;
+  listTitle: string;
+  uiPartTitleForTracking: string;
+  books: Book[];
```

Comment:
컴포넌트 본문에 이미 null check가 있으니 `books`를 무조건 받을 필요는 없습니다.

```suggestion
  books?: Book[];
```